src/radical/saga/adaptors/noop/noop_job.py

```diff
@@ -0,0 +1,683 @@
+
+__author__    = "Andre Merzky"
+__copyright__ = "Copyright 2020, The SAGA Project"
+__license__   = "MIT"
+
+
+''' noop based job adaptor implementation '''
```

Comment:
Should we use: no operation (noop)? Just trying to improve readibility but really non fundamental (who did not do some assembly in h[er|is] life?! :D

Reply:
hehe - done.